changelog/bugfixes/2022-05-22-bootengine.md

```diff
@@ -0,0 +1 @@
+- Fixed OEM ID from `pxe` to `metal` to follow the Ignition upgrade ([bootengine#45](https://github.com/flatcar-linux/bootengine/pull/45))
```

Comment:
```suggestion
- Fixed Ignition's OEM ID to be `metal` to follow the Ignition upstream change which otherwise resulted in a broken boot when the Flatcar OEM ID `pxe` was used ([bootengine#45](https://github.com/flatcar-linux/bootengine/pull/45))
```

Reply:
Thanks - reworded.